### PR TITLE
Coverage

### PR DESCRIPTION
**Pull request by ironspecs**
Add coverage reporting.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+  "freeze":     true,
+  "immed":      true,
+  "indent":     2,
+  "newcap":     true,
+  "noarg":      true,
+  "undef":      true,
+  "unused":     true,
+  "strict":     true,
+  "node":       true,
+  "latedef":    true,
```
Comment (nelsonpecora) [inline .jshintrc]: why did you remove the comments?

Comment (ironspecs) [inline .jshintrc]: Because comments are not valid json, and a jsonlinter gets angry at it.  Don't you have a jsonlinter in sublime?  :p

Also, all the comments were doing is repeating what the setting was.  Writing 2, and then two again in longer English, isn't really what comments are for.  If they were saying _why_ it's 2, then I'd be much more interested.

Comment (nelsonpecora) [inline .jshintrc]: yeah, but a .jshintrc technically isn't json.

I agree that "how" comments are bad and wrong, but some of them are "why" comments. Also, I really like knowing what `eqeqeq` and `newcap` do without having to look them up every. single. time.

(this is especially important with jshint, since half the rules are positive (allow these things) and half the rules are negative (ban these other things).

**Comment by nelsonpecora** [comment]
Unknown, eh? That's pretty spooky.

**Comment by ironspecs** [comment]
That is spooky.  At least its reporting coverage correctly.  I'll add back the comments.  Maybe someday they'll allow yaml.

**Comment by nelsonpecora** [comment]
[Now I have this stuck in my head](https://www.youtube.com/watch?v=l_iRdgGZ6Xc), thanks.
